coderd/provisionerjobs.go

```diff
@@ -101,7 +101,7 @@ func (api *API) handleAuthAndFetchProvisionerJobs(rw http.ResponseWriter, r *htt
 
 	qp := r.URL.Query()
 	p := httpapi.NewQueryParamParser()
-	limit := p.PositiveInt32(qp, 0, "limit")
+	limit := p.PositiveInt32(qp, 50, "limit")
```

Comment:
nit: perhaps a quick integration test to check if the limit is restricted :) ?

Reply:
We have a test that limit is applied, but are you suggesting we add a test that a default limit is applied as well?

Reply:
Yes. Just in case somebody accidentally changes it back to 0 -> heavy API responses.

Reply:
Done 5c917d3df9c8e76eb91d8193eebd171ffcf01fe2